todo_app/view_model.py

```diff
@@ -0,0 +1,21 @@
+from typing import Dict
+
+from todo_app.data.items_backend import Item, ItemsBackend, Status
+
+
+class ViewModel:
+    def __init__(self, items_backend: ItemsBackend) -> None:
+        self.items_backend = items_backend
+        self.Status = Status  # so the enum can be accessed from the template - imports cannot be used
+
+    def get_item_backend_name(self) -> str:
+        return type(self.items_backend).__name__
```

Comment:
Would it be simpler & nicer to ask each backend to provide a human-readable name?